cscs-checks/apps/jupyter/check_ipcmagic.py

```diff
@@ -24,38 +24,54 @@ class IPCMagicCheck(rfm.RunOnlyRegressionTest):
     executable_opts = ['tf-hvd-sgd-ipc-tf2.py']
     reference = {
         'daint:gpu': {
-            'slope': (2.0, -0.1, 0.1, None),
-            'offset': (0.0, -0.1, 0.1, None),
-            'retries': (0, None, None, None),
+            'slope': (2.0, -0.1, 0.1, 'N/A'),
+            'offset': (0.0, -0.1, 0.1, 'N/A'),
+            'retries': (0, None, None, 'N/A'),
```

Comment:
I would probably omit the units here altogether.

Reply:
They type checking seems to not allow mixing 3-tuples with 4-tuples

Reply:
ok, I see.